dnf-behave-tests/dnf/install-xml-base.feature

```diff
@@ -50,9 +50,9 @@ Given I make packages from repository "dnf-ci-fedora" accessible via http
   And I copy repository "dnf-ci-fedora" for modification
   And I generate repodata for repository "dnf-ci-fedora" with extra arguments "--baseurl http://localhost:{context.dnf.ports[dnf-ci-fedora]}"
   And I use repository "dnf-ci-fedora"
-  And file "/{context.dnf.repos[dnf-ci-fedora].path}/noarch/setup-2.12.1-1.fc29.noarch.rpm" exists
+  And file "/{context.dnf.repos[dnf-ci-fedora].path}/noarch/setup-0:2.12.1-1.fc29.noarch.rpm" exists
  When I execute dnf with args "install setup"
- Then file "/{context.dnf.repos[dnf-ci-fedora].path}/noarch/setup-2.12.1-1.fc29.noarch.rpm" exists
+ Then file "/{context.dnf.repos[dnf-ci-fedora].path}/noarch/setup-0:2.12.1-1.fc29.noarch.rpm" exists
```

Comment:
This is wrong, this step just matches a file in the filesystem, it doesn't use dnf at all.
Rpm files don't have the 0 epoch.